src/Ast/AST.php

```diff
@@ -0,0 +1,90 @@
+<?php declare(strict_types=1);
```

Comment:
I'd prefer the `declare(strict_types=1)` to be on its own line and take place after the header. [here](https://github.com/googleapis/google-auth-library-php/blob/v2.x/src/Auth/GoogleAuth.php#L18) is an example.

Reply:
Done.
I think all files currently have it on the first line; I'll do a global change in a future PR.